frontend/web/components/EditPermissions.tsx

```diff
@@ -285,23 +297,23 @@ const _EditPermissionsModal: FC<EditPermissionModalType> = forwardRef(
     const { data: organisationPermissions, isLoading: organisationIsLoading } =
       useGetRoleOrganisationPermissionsQuery(
         {
-          organisation_id: role?.organisation,
-          role_id: role?.id,
+          organisation_id: parseInt(`${role?.organisation}`),
+          role_id: parseInt(`${role?.id}`),
         },
         { skip: !role || level !== 'organisation' },
       )
 
     const { data: projectPermissions, isLoading: projectIsLoading } =
       useGetRoleProjectPermissionsQuery(
         {
-          organisation_id: role?.organisation,
-          project_id: id,
-          role_id: role?.id,
+          organisation_id: parseInt(`${role?.organisation}`),
+          project_id: parseInt(`${id}`),
```

Comment:
Just double checking that this line is right. The project id was set globally as `id`?

Reply:
id is a prop of EditPermissions, it's the id of the corresponding entity that we're editing the permission of